cibuildwheel/__main__.py

```diff
@@ -197,19 +199,15 @@ def _compute_platform_ci() -> PlatformName:
             file=sys.stderr,
         )
         sys.exit(2)
-    if sys.platform.startswith("linux"):
-        return "linux"
-    elif sys.platform == "darwin":
-        return "macos"
-    elif sys.platform == "win32":
```

Comment:
This used to say `sys.platform == "win32"` but this changes it to `startswith("win")`. Am I introducing a bug with this behavior change? If so I'd like to add a comment explaining why we use `== "win32"` here but in architecture we use `startswith("win")`.